api/docker-compose.yaml

```diff
@@ -25,6 +25,7 @@ services:
         hard: -1
     volumes:
       - dojo_esdata:/usr/share/elasticsearch/data
+      - ./big_models_data_cache:/api/big_models_data_cache
```

Comment:
Second important change. We could decide to move to docker-compose-dev.yaml but I don't have enough information on how/if we use that yet.

Reply:
@mattprintz I'm hoping you have both context on the docker-compose-dev file as well as opinion on if we should only use this caching mechanism on dev vs always (including prod). I'm of the opinion that we should do it both during local dev and in deployments.

Reply:
docker-compose-dev isn't really used anywhere any longer. For "dev" purposes, the changes should be in the main `docker-compose.build-override.yaml` file.

The "local" docker-compose files (the ones in the sub directories) are the docker compose rules if we were going to be using docker compose to run just that service by itself, like if we had a multi-server deploy with api on one server and terminal on another, etc.

Reply:
I think this change is fine for both prod and dev.